src/Rx/Weno/NewRxData.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2019. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+require_once "../../interface/globals.php";
```

Comment:
remove above line

Reply:
If I remove the globals line the next line won't work.

Reply:
Trust me. It will still work :)
(this class is used in transmitrx.php which has already required globals.php)